sdcm/sct_config.py

```diff
@@ -95,6 +95,9 @@ class SCTConfiguration(dict):
         dict(name="n_db_nodes", env="SCT_N_DB_NODES",  type=int_or_list,
              help="Number list of database nodes in multiple data centers."),
 
+        dict(name="n_ot_nodes", env="SCT_N_OT_NODES",  type=int_or_list,
```

Comment:
Name is not clear for someone that reads it.
please change it to: n_gemini_test_oracle_db_nodes

Reply:
Done